src/transformStateWithClones.js

```diff
@@ -65,7 +65,33 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, transforms) {
-  // write code here
+  const stateArr = [];
+  const transformationState = { ...state };
```

Comment:
Лучше не писать Arr в переменных.
```suggestion
  const transformedObjects = [];
  const transformedObject = { ...state };
```